pypeit/spectrographs/ldt_deveny.py

```diff
@@ -113,6 +113,7 @@ def init_meta(self):
         # Extras for config and frametyping
         self.meta['idname'] = dict(ext=0, card='IMAGETYP')
         self.meta['dispangle'] = dict(ext=0, card='GRANGLE', rtol=1e-3)
+        self.meta['cenwave'] = dict(card=None, compound=True, rtol=5)
```

Comment:
Does 5 here mean 5Ang?  Seems a touch big if so
unless the resolution is very low..